moveit_core/robot_trajectory/src/robot_trajectory.cpp

```diff
@@ -258,27 +255,24 @@ void RobotTrajectory::getRobotTrajectoryMsg(moveit_msgs::msg::RobotTrajectory& t
   if (!onedof.empty())
   {
     trajectory.joint_trajectory.header.frame_id = robot_model_->getModelFrame();
-    trajectory.joint_trajectory.header.stamp = stamp;
+    trajectory.joint_trajectory.header.stamp = rclcpp::Time(0, 0, RCL_ROS_TIME);
```

Comment:
Looking at the documentation for rclcpp::Time, it looks like the default is RCL_SYSTEM_TIME. What's the reason for something different here?

Reply:
RCL_ROS_TIME is default for node time as it should be able to support simulation and is mostly used for messages.